src/encrypted_models.rs

```diff
@@ -293,7 +293,7 @@ pub enum CollectionAccessLevel {
 
 #[derive(Serialize, Deserialize, Clone)]
 #[serde(rename_all = "camelCase")]
-pub struct EncryptedCollection {
+pub(crate) struct EncryptedCollection {
```

Comment:
I'm pretty sure this is exposed to users of this library.

Reply:
Same here, it it was exposed, `pub(crate)` wouldn't compile.

Reply:
OK.